Platform/ElkhartlakeBoardPkg/CfgData/CfgData_Silicon.yaml

```diff
@@ -1857,21 +1857,43 @@
       option       : $EN_DIS
       help         : >
                      When FALSE, it disables Timed GPIO 0.
-      length       : 1b
+      length       : 0x01
       value        : 0x0
   - EnableTimedGpio1 :
       name         : Timed GPIO 1
       type         : Combo
       option       : $EN_DIS
       help         : >
                      When FALSE, it disables Timed GPIO 1.
-      length       : 1b
+      length       : 0x01

```

Comment:
Why do you want to use 1 byte  instead of 1 bit for ENABLE/DISABLE?

Reply:
ok i misunderstood the 1b ad byte and 0x1 as bit. Will update the patch